bot/cogs/cyber.py

```diff
@@ -12,6 +12,19 @@
 from bot.constants import BASE_ALIASES, CYBERDISC_ICON_URL, HINTS_LIMIT, PWNED_ICON_URL
 
 
+async def generatebase64(seed: int):
+    import random
+    import string
+    random.seed(seed)
+    letters = string.ascii_letters + string.digits + "+" + "/" + "="
+    result = ""
+
+    for _ in range(20):
```

Comment:
You can use the `k` parameter of `random.choices` without needing to use a for loop:
![image](https://user-images.githubusercontent.com/25161069/51548301-fc972100-1e5f-11e9-924c-71afe104ebfe.png)